### ISSUE TITLE
32p logins failing

### ISSUE THREAD

**Issue by bonebox** [evilhero/mylar#2134]
32p logins are failing authentication with the latest development release. It was working previously and manual logins are successful.

**Comment by bonebox** [comment]
I initially noticed several entries in the Mylar log about site maintenance, but it still allowed me to log in manually immediately after those errors appeared.  Now it appears to be failing because 32p is now requiring a captcha due to the prior failed auth attempts from Mylar.

**Comment by bonebox** [comment]
Also, Mylar says it's disabling further login attempts, but it still attempts when a series/issue is marked as wanted. It's now locked me out for 6 hours.

**Comment by bonebox** [comment]
I started using Mylar again a month or so ago for the first time in over a year, so not sure about the cookie, and haven't changed my 32p pw in quite some time. It's been authenticating fine until yesterday when I updated Mylar, but sounds like that was just a coincidence.  Regardless, I've disabled 32p in Mylar for now to prevent it from prolonging the lockout and will try again once the captcha is lifted and let you know. Thanks!
